misk/src/main/kotlin/misk/digester/Windower.kt

```diff
@@ -0,0 +1,108 @@
+package misk.digester
+import java.util.Calendar
+/** Window represents a time range */
+class Window (private val start: Calendar, private val end: Calendar) {
```

Comment:
Java time libraries have changed a lot over the years. The preferred current way of representing instants in time is `java.time.Instant`